### DISCUSSION TITLE
Livewire Calendar in the wild

### DISCUSSION THREAD

**Discussion by joshtorres** [omnia-digital/livewire-calendar#22] [Show and tell]
Show us where you are using livewire calendar by posting a url and/or image!

**Comment by lrljoe** [comment]
![image](https://github.com/user-attachments/assets/9ee7cb5a-5810-4f44-b34e-3eb32b32087f)
Nice, simply, spanning and repeated events.

Yes, I know, it needs cleaning up to make it look prettier!
